upgrade.yml

```diff
@@ -416,6 +416,11 @@
   any_errors_fatal: true
   tags: swift
 
+  pre_tasks:
+    - name: stop swift services started by swift-init
+      command: swift-init stop all
+      ignore_errors: True
```

Comment:
It affects the availability of cluster during 2.x to 3.0 upgrade. Swift should be able to recover since we are not modifying rings. We can always add "serial" to mitigate